test/suites/integration/backend/upload-download-file.test.ts

```diff
@@ -268,5 +272,36 @@ await describeMspNet(
         "Root of bucket should have changed"
       );
     });
+
+    it("Should successfully download a file via the backend API", async () => {
+      // Ensure the upload test completed successfully
+      assert(uploadedFileKeyHex, "Upload test must complete successfully before download test");
+      assert(originalFileBuffer, "Original file buffer must be available from upload test");
+
+      const response = await fetch(`http://localhost:8080/download/${uploadedFileKeyHex}`);
```

Comment:
I'd also include the authentication header.  The endpoint doesn't have it but it should, it was just removed during development, so I'm of the idea that we should still have the header here, so when it does become necessary it's easier to see what needs to change